paper/user_study.tex

```diff
@@ -49,8 +49,128 @@ \subsection{Results}
+        \cline{2-3}
+        & 6. In the schema editor, it would be more consistent and user-friendly if the functionality to give a name to a property were located on the right side of the GUI editor panel, eliminating the need to click on the property first in order to rename it. & 6. We have retained the property naming on the right end but improved it with an intuitive design. When clicking on a new property in the schema editor, it is now highlighted to prompt the user to change the name. \\
+        \cline{2-3}
+        & 7. The search function for locating specific properties lacks clarity at first glance. It should provide an immediate response and extend to nested levels, rather than merely highlighting the higher-level property in yellow. & 7. The search functionality has been enhanced. It now provides a list of properties containing the keyword, and upon clicking, it highlights the selected property while opening the nested levels in the GUI editor panel for improved usability. \\
+        \hline
+    \end{tabular}
+    \caption*{User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+\begin{table*}
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 3 & 1.Still I have to right & 1. bla bla \\
+        \cline{2-3}
+
+        \hline
+    \end{tabular}
+    \caption*{User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+\begin{table*}
+
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 4 & 1.Modifying or renaming a 'new property' in Schema Editor does not appear to take effect when double-clicking on it. & 1. Renaming of the new property is made possible by double-clicking on 'newProperty' itself. \\
+        \cline{2-3}
+        & 2. Upon selecting a property type in the schema editor, the type of the child property should be automatically chnaged. & 2. The type of the child element is automatically adjusted according to parent type. \\
+        \cline{2-3}
+        & 3.A toggle button should be implemented to enable and disable the code panel and GUI panel. & 3. Will be consider for the future work. \\
+        \cline{2-3}
+        & 4.When working with a single property in the File Editor within the GUI Editor panel, the opacity of already open properties should be decreased, rather than making everything visible. & 3. Will be consider for the future work. \\
```

Comment:
```suggestion
        & 4.When working with a single property in the File Editor within the GUI Editor panel, the opacity of already open properties should be decreased, rather than making everything visible. & 3. Will be considered in future work. \\
```